packages/api/tests/endpoints/granules/test-reingest-granules.js

```diff
@@ -69,31 +84,32 @@ test.after.always(async (t) => {
   });
 });
 
-test('put request with reingest action calls the granuleModel.reingest function with expected parameters', async (t) => {
-  const granuleGetStub = sinon.stub(models.Granule.prototype, 'get').returns(
-    new Promise((resolve) => resolve({ collectionId: fakeCollectionId }))
-  );
-  const granuleReingestStub = sinon.stub(models.Granule.prototype, 'reingest').returns(
-    new Promise((resolve) => resolve({ response: 'fakeResponse' }))
-  );
+test('put request with reingest action calls the reingestGranule function with expected parameters', async (t) => {
+  const {
+    granuleId,
+  } = t.context;
+
+  const granuleReingestStub = sinon.stub().resolves({ response: 'fakeResponse' });
 
   const body = {
     action: 'reingest',
   };
 
-  await request(app)
-    .put('/granules/asdf')
-    .set('Accept', 'application/json')
-    .set('Authorization', `Bearer ${jwtAuthToken}`)
-    .send(body)
-    .expect(200);
+  await put({
+    body,
+    params: {
+      granuleName: granuleId,
+    },
+    testContext: {
+      reingestHandler: granuleReingestStub,
+    },
+  },
+  buildFakeExpressResponse());
```

Comment:
Consider explicitly failing the test here if one of the sinon stubs is called

Reply:
So I was looking at this and I know I'm the one that called it out as an issue but it might not be. `put` should throw any errors:

```
} catch (error) {
      if (error instanceof RecordDoesNotExist) {
        return res.boom.BadRequest(`Cannot reingest granule: ${error.message}`);
      }
      throw error;
    }
```
And if that happens the test will fail with an error. I was thinking there was a path here that would result in `put` swallowing an error and us ignoring the response. I don't think that's possible though.

Reply:
Hmm, I think you're right for this test, *only* because *this* test doesn't have a logic path that results in a call like `return res.boom.<something>` without a throw *after* the `granuleReingestStub` is called.